core/streamdata.go

```diff
@@ -56,6 +56,7 @@ type SegTranscodingMetadata struct {
 	AuthToken        *net.AuthToken
 	DetectorEnabled  bool
 	DetectorProfiles []ffmpeg.DetectorProfile
+	CalcSign         bool
```

Comment:
Ah I see. At first, I was going to suggest setting the field in `ffmpeg.VideoProfile` since wherever `ffmpeg.TranscodeOptions` is used, the field can be read from the `ffmpeg.VideoProfile` field set on the `ffmpeg.TranscodeOptions`. But...a few reasons I can see for why setting the field in `ffmpeg.VideoProfile` might not be desirable:

- While we're calculating the phash per output, I don't currently see a scenario where we'd calculate the phash for one output, but not the others. So, if we had to set the field per video profile, that would be annoying when the value of the field should be the same for all video profiles
- Given that I think its preferable to avoid storing a per video profile field if it is always going to be same value for each profile, for https://github.com/livepeer/go-livepeer/issues/1993, I also think it'd be annoying having to set the field on JSON profiles just so we can convert them into `ffmpeg.VideoProfile` structs with that field set in the [helper](https://github.com/livepeer/go-livepeer/blob/92d274d3aae43c09f4106ca5f9621e3b80b101b4/server/mediaserver.go#L374)

Given the above reasoning, I think keeping the field as-is set on `SegTranscodingMetadata` and its equivalent structs (i.e. protobuf message) is reasonable.